<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battle</class>
 <widget class="QWidget" name="Battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip {
	opacity: 100;
	font: 75 16pt &quot;Consolas&quot;;			
	background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="myPoke" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QLabel" name="myPokeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="enemyPoke" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QLabel" name="enemyLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="myPokeHead" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>99</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLabel" name="myPokeInfoLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="myPokeHPWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <widget class="QProgressBar" name="myHPBar">
     <property name="geometry">
      <rect>
       <x>-3</x>
       <y>0</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="enemyPokeHead" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLabel" name="enemyPokeInfoLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="enemyPokeHPWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <widget class="QProgressBar" name="enemyHPBar">
     <property name="geometry">
      <rect>
       <x>-3</x>
       <y>0</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
